lib/moon_web/components/themes_select/theme_switcher.ex

```diff
@@ -3,24 +3,20 @@ defmodule MoonWeb.Components.ThemesSelect.ThemeSwitcher do
 
   use Moon.StatelessComponent
 
-  alias Moon.Components.Form
-  alias Moon.Design.Switch
-  alias Moon.Components.Field
-  alias MoonWeb.Components.ThemesSelect.SelectedTheme
+  alias Moon.Icons.OtherMoon
```

Comment:
again - do we still need this component at all?
is it used anywhere?

Reply:
Not needed. I had deleted this component, but probably reverted the change by accident and didn't notice.